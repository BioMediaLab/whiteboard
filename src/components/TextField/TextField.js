import { TextField } from '@shopify/polaris'

export default TextField
