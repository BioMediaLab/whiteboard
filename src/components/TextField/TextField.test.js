import React from 'react'
import TextField from './'

describe('TextField', () => {
  test('can render', () => {
    const component = render(
      <TextField
        id="example"
        name="example"
        label="example"
        value="woohoo!"
        onChange={() => {}}
      />
    )

    expect(component).toMatchSnapshot()
  })
})
