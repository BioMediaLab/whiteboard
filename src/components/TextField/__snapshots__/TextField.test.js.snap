// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField can render 1`] = `
<div>
  <div
    class="TextField"
  >
    <label
      for="example"
    >
      example
    </label>
    <input
      id="example"
      name="example"
      type="text"
      value="woohoo!"
    />
  </div>
</div>
`;
